src/functional/iterator/embedded.py

```diff
@@ -44,10 +41,9 @@
 FieldIndexOrIndices: TypeAlias = FieldIndex | tuple[FieldIndex, ...]
 
 FieldAxis: TypeAlias = (
-    Dimension | Offset
+    common.Dimension | runtime.Offset
 )  # TODO Offset should be removed, is sometimes used for sparse dimensions
-TupleAxis: TypeAlias = NoneType
```

Comment:
It was here because specifying tuple dimensions as `None` might not be the right long term choice. Therefore, it was increased to improve readability. I'd prefer rolling back all the changes regarding `TupleAxis`, but I'd like to hear your thoughts about it.

Reply:
I did not delete this for fun. Mypy was specifically complaining about `NoneType` and saying to replace it with `None`. But since it is used as a type later, that caused problems because `None` is not a type.

If there is a reason for `TupleAxis` to be there, the solution is probably a type ignore comment.